aten/src/ATen/core/Error.cpp

```diff
@@ -108,4 +108,20 @@ std::string GetExceptionString(const std::exception& e) {
 #endif // __GXX_RTTI
 }
 
+std::function<std::string(void)>* GetFetchStackTrace() {
```

Comment:
Sorry for the late comment, but why this? It seems that this is a dummy function that only returns an empty string, so the at::ThrowEnforceNotMet is essentially a no-function. In this case, we should not use this macro because it is in fact a redundant version of AT_ASSERTM.

Reply:
I copied it from caffe2 side, I was thinking the same! But I didn't want to break the old behavior at all for this PR. (Was actually expecting this discussion 😄)